cmd/cloud_sql_proxy/proxy.go

```diff
@@ -305,15 +307,21 @@ func parseTCPOpts(ntwk, addrOpt string) (string, error) {
 // parseInstanceConfigs calls parseInstanceConfig for each instance in the
 // provided slice, collecting errors along the way. There may be valid
 // instanceConfigs returned even if there's an error.
-func parseInstanceConfigs(dir string, instances []string, cl *http.Client) ([]instanceConfig, error) {
+func parseInstanceConfigs(dir string, instances []string, cl *http.Client, skipFailedInstanceConfigs bool) ([]instanceConfig, error) {
 	errs := new(bytes.Buffer)
 	var cfg []instanceConfig
 	for _, v := range instances {
 		if v == "" {
 			continue
 		}
 		if c, err := parseInstanceConfig(dir, v, cl); err != nil {
-			fmt.Fprintf(errs, "\n\t%v", err)
+			if skipFailedInstanceConfigs == true {
+				logging.Infof("There was a problem when parsing a instance configuration but ignoring due to the configuration. Error: %v", err)
+				// we intentionally don't add to `errs` since we don't want to bubble these
```

Comment:
nit: This comment can probably be left out. 

At the very least, it should be properly capitalized/punctuated - see [Comment Sentences](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences).